server/app.js

```diff
@@ -7,4 +7,6 @@ app.use(express.static(path.join(__dirname, '../dist')));
 
 app.use(morgan('dev'));
 
+app.use('/api', require('./routes/api'));
```

Comment:
Why is the `require` inline?

Reply:
Most linting setups should scream at this.

Reply:
Fixed.  Linter didn't even whisper.  Looking to add a rule, but not sure what to add when looking at rules here:  [https://eslint.org/docs/rules/](url)